c++/src/CMakeLists.txt

```diff
@@ -184,13 +184,21 @@ set(SOURCE_FILES
   Timezone.cc
   TypeImpl.cc
   Vector.cc
-  Writer.cc)
+  Writer.cc
+  CpuInfoUtil.cc
+  BpackingDefault.cc)
```

Comment:
Please sort them alphabetically.

Reply:
Why `CpuInfoUtil.cc` is always required?

Reply:
Already sort them alphabetically.

The reason for CpuInfoUtil.cc building default is 
[RleDecoderV2.cc] readLongs => [Dispatch.hh] DynamicDispatch => Resolve
The function Resolve uses CpuInfo to judge if the CPU supports the current Bit-unpacking function.
So I feel that keeping CpuInfoUtil.cc in the default building source file list, maybe better.